### PR TITLE
Try out the pdf-preview without pagedjs

### PR DESCRIPTION
**Pull request by frozenhelium**

### CODE REVIEW

File: static/pdf-preview-styles.css [hunk 1]
```diff
@@ -10,6 +9,10 @@
   }
 }
 
+img {
```
Comment (batpad) [inline static/pdf-preview-styles.css:12]: this looks a bit dangerous? what about images that have a lower intrinsic height? They'd maybe get stretched? And I assume this is trying to solve the problem of not knowing image sizes? Can we instead figure out firing a callback when all images have loaded or so?

Not a blocker! But let's chat about this one.

Comment (frozenhelium) [reply static/pdf-preview-styles.css:12]: @batpad this file is not currently being used. It was previously used by the pagedjs but not anymore. I'll clean-up this file and other pagedjs related stuff once we are sure about this method

Comment (batpad) [reply static/pdf-preview-styles.css:12]: Ah, I see this is not actually being used any more 👍

**Review by wrynearson — APPROVED** [review]
Thank you @frozenhelium!!

I tested three ATBDs with a variety. All content is present! 🥳 

Page breaks handle images well. I did see captions being bumped to the next page in one case, but I think this is acceptable.

image

I also saw text in tables being indented, which we can address in another PR:

image

For future reference – this PR removes page numbers from the ToC. The ToC labels are still hyperlinked, linking to the correct sections of the PDF. Page numbers are also removed. We will look into adding them back using another method.

**Pull request merged**